lib/spring/application_manager.rb

```diff
@@ -42,7 +42,7 @@ def with_child
         if alive?
           begin
             yield
-          rescue Errno::ECONNRESET, Errno::EPIPE
+          rescue Errno::ECONNRESET, Errno::EPIPE, Errno::EINVAL
```

Comment:
Should we do some kind of cleanup here in case the child is hanging around as a zombie process?